hpedockerplugin/volume_manager.py

```diff
@@ -218,10 +209,18 @@ def create_snapshot(self, src_vol_name, snapshot_name,
         snap_qos = volume.DEFAULT_QOS
         mount_cononflict_delay = vol['mount_conflict_delay']
 
+        is_snap = True
         snap_vol = volume.createvol(snapshot_name, snap_size, snap_prov,
                                     snap_flash, snap_compression, snap_qos,
                                     mount_cononflict_delay, is_snap)
 
+        # TODO: Remove the below code or remove snapshot back reference entry
```

Comment:
@imran-ansari line #217 to #233 can be removed

Reply:
Sure